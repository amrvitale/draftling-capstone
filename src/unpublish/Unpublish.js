import React from 'react';

/* This component renders the Unpublish button, found on an individual draftling's page when the status is "published" */

class Unpublish extends React.Component {
    render() {
        return (
            <div>
                <button onClick={this.props.onClick}>
                    Unpublish
                </button>
            </div>
        );
    }
}

export default Unpublish;