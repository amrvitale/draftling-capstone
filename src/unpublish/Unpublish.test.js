import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Unpublish from './Unpublish';

describe (`Unpublish component`, () => {
    it('renders a .Unpublish by default', () => {
        const wrapper = shallow(<Unpublish />)
        expect (toJson(wrapper)).toMatchSnapshot()
    })

    
})