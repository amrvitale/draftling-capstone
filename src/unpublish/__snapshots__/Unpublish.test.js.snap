// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unpublish component renders a .Unpublish by default 1`] = `
<div>
  <button>
    Unpublish
  </button>
</div>
`;
